src/aphroditeInterfaceFactory.js

```diff
@@ -1,18 +1,37 @@
 import { flushToStyleTag } from 'aphrodite/lib/inject';
+import rtlCSSJS from 'rtl-css-js';
 
-import resolve from './utils/resolve';
+import resolveLTR from './utils/resolveLTR';
+import resolveRTL from './utils/resolveRTL';
 
 export default ({ StyleSheet, css }/* aphrodite */) => ({
   create(styleHash) {
+    return this.createLTR(styleHash);
+  },
+
+  createLTR(styleHash) {
     return StyleSheet.create(styleHash);
   },
 
+  createRTL(styleHash) {
+    const styleHashRTL = {};
+    Object.entries(styleHash).forEach(([styleKey, styleDef]) => {
```

Comment:
to confirm, do we need/have a shim for `Object.entries`?

Reply:
@ljharb would know. 

I didn't think so? We were using it before.

Reply:
We should probably use https://npmjs.com/object.entries; otherwise we have an implicit dependency on people using https://npmjs.com/airbnb-js-shims (which isn't terrible, but isn't ideal).

Reply:
Sounds good. I can add that.